admin_guide/topics/creating-LC-and-ASG-cluster-auto-scaler.adoc

```diff
@@ -90,14 +90,14 @@ NOTE: If your template is fewer than 16 KB before you encode it, you can provide
 $ aws autoscaling create-auto-scaling-group \
       --auto-scaling-group-name mycluster-ASG \ <1>
       --launch-configuration-name mycluster-LC \ <2>
-      --min-size 0 \ <3>
+      --min-size 1 \ <3>
       --max-size 6 \ <4>
       --vpc-zone-identifier subnet-12345678 \ <5>
       --tags ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=Name,Value=mycluster-ASG-node,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=kubernetes.io/cluster/mycluster,Value=true,PropagateAtLaunch=true ResourceId=mycluster-ASG,ResourceType=auto-scaling-group,Key=k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/compute,Value=true,PropagateAtLaunch=true <6>
 ----
 <1> Specify the name of the Auto Scaling group, which you use when you deploy the auto-scaler deployment
 <2> Specify the name of the Launch Configuration that you created.
-<3> Specify the minimum number of nodes that the auto-scaler maintains.
+<3> Specify the minimum number of nodes that the auto-scaler maintains. At least one node is required for the template to function.
 <4> Specify the maximum number of nodes the scale group can expand to.
 <5> Specify the VPC subnet-id, which is the same subnet that the cluster uses.
-<6> Specify this string to ensure that Auto Scaling group tags are propagated to the nodes when they launch.
```

Comment:
Can you please check why this line is being listed as changed? I can't see any differences here. Thank you!

Reply:
Hi, Paul. I didn't deliberately make a change here. Perhaps I accidentally hit a key and didn't undo it properly. This line should be fine as is.

Reply:
Ok thank you. Just for reference, looking at `git show` for your commit, I think the change relates to a new line having been added to the end of the file:

```
-<6> Specify this string to ensure that Auto Scaling group tags are propagated to the nodes when they launch.
\ No newline at end of file
+<6> Specify this string to ensure that Auto Scaling group tags are propagated to the nodes when they launch.
```